test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -24,6 +26,9 @@ public static void Main ()
 			TestRequiresSuppressesWarningsFromReflectionAnalysis ();
 			TestDuplicateRequiresAttribute ();
 			TestRequiresUnreferencedCodeOnlyThroughReflection ();
+			TestVirtualMethodRequiresUnreferencedCode ();
+			TestStaticCctorRequiresUnreferencedCode ();
+			TestDynamicallyAccessedMembersWithRequiresUnreferencedCode (typeof (BaseType));
```

Comment:
This should use the `DynamicallyAccessedTypeWithRequiresUnreferencedCode` type as the parameter and it should produce a warning on the .ctor right?